server/plugin/kv.go

```diff
@@ -85,6 +82,43 @@ func (s *pluginStore) DeleteUser(mattermostUserID string) error {
 	return nil
 }
 
+func (s *pluginStore) GetAllUsers() ([]*serializer.IncidentCaller, error) {
+	page := 0
+	users := []*serializer.IncidentCaller{}
+	for {
+		kvList, err := s.plugin.API.KVList(page, constants.DefaultPerPage)
+		if err != nil {
+			return nil, err
+		}
+
+		if len(kvList) == 0 {
+			return users, nil
+		}
+
+		for _, key := range kvList {
+			if userID, isValidUserKey := IsValidUserKey(key); isValidUserKey {
+				decodedKey, decordErr := decodeKey(userID)
+				if decordErr != nil {
+					return nil, decordErr
+				}
+
+				user, loadErr := s.LoadUser(decodedKey)
+				if loadErr != nil {
+					return nil, loadErr
+				}
+
+				users = append(users, &serializer.IncidentCaller{
+					MattermostUserID: user.MattermostUserID,
+					Username:         user.Username,
+					ServiceNowUser:   user.ServiceNowUser,
+				})
+			}
+		}
+
+		page++
```

Comment:
```suggestion
if len(kvList) < constants.DefaultPerPage {
    break
}
		page++
```
This will save one extra iteration.